actions/image_partition_action.go

```diff
@@ -151,6 +155,7 @@ import (
 type Partition struct {
 	number   int
 	Name     string
+	Label    string
```

Comment:
I don't get what this adds; What does it do you cannot currently do?

Reply:
currently the name parameter is checked for uniqueness; adding an optional label parameter would allow the partitions to have the same label or a different label to the name.

for instance,
```
- name: rootfs-A
  label: rootfs
  partlabel: rootfs

- name: rootfs-B
   label: rootfs
   partlabel: rootfs
```

Can drop the patch if you don't see the benefit to it

Reply:
dropped patch